### ISSUE TITLE
TikTok blocks this request displaying a Captcha

### ISSUE THREAD

**Issue by Mayaman5566** [snlbaral/tiktok-dl#9]
Some errors in tiktok.py

TikTok blocks this request displaying a Captcha 
Tip: Consider using a proxy or a custom_verifyFp as method parameters

**Comment by snlbaral** [comment]
yes there is that issue, currently no exact working solution for every time. try using tiktok-auto.py as of now.

**Comment by hacknapier** [comment]
TikTok Captcha Solving Service is available here with demo to Login successfully into your TikTok Account.
https://rapidapi.com/flyCaptcha/api/flycaptcha
